packages/ripple-binary-codec/src/types/blob.ts

```diff
@@ -33,6 +33,9 @@ class Blob extends SerializedType {
     }
 
     if (typeof value === 'string') {
+      if (!/^[A-F0-9]*$/iu.test(value)) {
+        throw new Error('Cannot construct Bob from a non-hex string')
```

Comment:
I don't think we'd want to construct Bob 👷‍♂️ from a non-hex string
```suggestion
        throw new Error('Cannot construct Blob from a non-hex string')
```

Reply:
🤣